ansible/roles/vm_set/tasks/add_ceos_list.yml

```diff
@@ -98,12 +98,30 @@
 
 - name: Create VMs network
   become: yes
+  loop: "{{ VM_targets|flatten(levels=1) }}"
+  loop_control:
```

Comment:
I think we should add an option to enable this behavior instead of enabling this for all testbeds by default.

Reply:
> I think we should add an option to enable this behavior instead of enabling this for all testbeds by default.

We use the parameter `enable_async` to control this feature.

Reply:
> also options to control the number of parallel tasks. :D

We limit the number to 50.